src/server/game/Entities/Player/Player.cpp

```diff
@@ -27087,6 +27360,8 @@ void Player::OnCombatExit()
 {
     UpdatePotionCooldown();
     m_combatExitTime = getMSTime();
+    if (!IsInAreaThatActivatesPvpTalents())
```

Comment:
This is wrong - pvp talents remain active for 20 seconds after leaving combat (tied to aura 134735)